brands/shop/mall.json

```diff
@@ -0,0 +1,11 @@
+{
+  "shop/mall|Westfield": {
+    "countryCodes": ["au"],
+    "tags": {
+      "brand": "Westfield",
+      "brand:wikidata": "Q2036076",
+      "brand:wikipedia": "en:Westfield Group",
```

Comment:
This article and Wikidata item pertain to a former company (two companies ago, actually). The closest thing left is [this disambiguation page of lists of malls](https://en.wikipedia.org/wiki/List_of_Westfield_shopping_centres), which points to the need for a separate Wikidata item covering the brand that’s in use by two different companies (one in 🇦🇺🇳🇿, the other in 🇬🇧🇺🇸).

Reply:
→ #2743

Reply:
Ok thanks, I'm happy to just remove the brand:wikidata and brand:wikipedia tags then. Should I do a new PR which removes them as a quick fix?

Reply:
I just removed them in ea284122